flowauth/backend/tests/conftest.py

```diff
@@ -69,40 +75,31 @@ def test_user(app):
 @pytest.fixture
 def test_admin(app):
     with app.app_context():
-        user = User(
-            username="TEST_ADMIN", password="TEST_ADMIN_PASSWORD", is_admin=True
-        )
-        ug = Group(name="TEST_ADMIN", user_group=True, members=[user])
-        db.session.add(user)
-        db.session.add(ug)
+        user = User.query.filter(User.username == app.config["ADMIN_USER"]).first()
+
+        return user.id, user.username, app.config["ADMIN_PASSWORD"]
+
+
+@pytest.fixture
+def test_group(app):
+    with app.app_context():
+        group = Group(name="TEST_GROUP")
+        db.session.add(group)
         db.session.commit()
-        return user.id, user.username, "TEST_ADMIN_PASSWORD"
+        return group.id, group.name
```

Comment:
Can we return a namedtuple here (or something similar)? Otherwise the tests below are a bit cryptic (what does `test_group[0]` mean?)